opentelemetry-api/opentelemetry/backend.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2019, OpenTelemetry Authors
```

Comment:
I haven't tested, would the `backend.py` file under `opentelemetry` folder break the namespace package?

Reply:
I will test this on my side.

Reply:
I think that should work:

> Namespace packages allow you to split the sub-packages **and modules** within a single package across multiple, separate distribution packages

-- https://packaging.python.org/guides/packaging-namespace-packages/

Reply:
It worked after adding `*.py` to the namespace discover rule in `setup.py`:

```python
    packages=setuptools.find_namespace_packages(include=["*.py", "opentelemetry.*"]),
```

Reply:
I'm resolving this, since we now settled for the `src/` approach.